deploy/helm/sumologic/README.md

```diff
@@ -82,6 +82,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
+| `fluentd.metadata.cache_refresh_apiserver_request_delay`                  | Option to control the delay with which cache refresh calls hit the api server.For example, if 0 then all metadata enrichment happen immediately. Setting this to a non-zero values ensures the traffic to api server is much distributed.                                                                                                                                                                                                     | `0`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
`server.For` - missing a space here.

`traffic to api server is much distributed` - I think you meant `more distributed`?